src/context.ts

```diff
@@ -33,15 +33,29 @@ const dbFile = path.join(
   'database.sqlite',
 );
 
-const Context: ContextInterface = {
+interface SessionStorageFactory {
+  SESSION_STORAGE_: SessionStorage | (() => SessionStorage);
+}
+
+const Context: ContextInterface & SessionStorageFactory = {
   API_KEY: '',
   API_SECRET_KEY: '',
   SCOPES: new AuthScopes([]),
   HOST_NAME: '',
   API_VERSION: ApiVersion.Unstable,
   IS_EMBEDDED_APP: true,
   IS_PRIVATE_APP: false,
-  SESSION_STORAGE: new SQLiteSessionStorage(dbFile),
+
+  SESSION_STORAGE_: () => new SQLiteSessionStorage(dbFile),
```

Comment:
What if instead of doing this here and adding the getter / setter, we only changed `initialize` (i.e. `ContextParams`) to take in a function and default this to a null value (or memory storage), defaulting to SQLite in initialize?

Reply:
That is probably the better suggestion, yes. I’ll change that